graylog2-web-interface/package.json

```diff
@@ -10,10 +10,10 @@
   },
   "readme": "../README.md",
   "scripts": {
-    "start": "./node_modules/.bin/webpack --config webpack.vendor.js && ./node_modules/.bin/webpack-dev-server --config webpack.combined.config.js",
-    "start-nohmr": "./node_modules/.bin/webpack --config webpack.vendor.js && ./node_modules/.bin/webpack-dev-server --watch --config webpack.combined.config.js",
-    "build": "./node_modules/.bin/webpack -p --config webpack.vendor.js && ./node_modules/.bin/webpack",
-    "test": "./node_modules/.bin/webpack --config webpack.vendor.js && ./node_modules/karma/bin/karma start karma.ci.conf.js",
```

Comment:
It doesn't use the globally installed webpack, see [the npm docs](https://docs.npmjs.com/misc/scripts#path). The scripts of installed packages are prepended to the path of scripts run by npm. BTW: I don't have webpack installed globally either.